tools/include/eosio/abigen.hpp

```diff
@@ -434,10 +434,10 @@ namespace eosio { namespace cdt {
 
          if(inside_type_name[0] != "" && inside_type_name[1] != ""){
             ret += inside_type_name[0] + "_" + inside_type_name[1];
-            abidef.type = "pair_" + inside_type_name[0] + "_" + inside_type_name[1] + "[]";
+            abidef.type = "mpair_" + inside_type_name[0] + "_" + inside_type_name[1] + "[]";
 
             abi_struct kv;
-            kv.name = "pair_" + inside_type_name[0] + "_" + inside_type_name[1];
+            kv.name = "mpair_" + inside_type_name[0] + "_" + inside_type_name[1];
```

Comment:
Can above kv.name be the same as abidef.type with +"[]" being added at the end?

Reply:
Yes, it will add [] if it is map.